src/test/scala/org/camunda/feel/impl/builtin/BuiltinConversionFunctionsTest.scala

```diff
@@ -63,6 +63,18 @@ class BuiltinConversionFunctionsTest
     )
   }
 
+  it should "return null if the date is not valid (leap year)" in {
+
+    evaluateExpression(""" date(x) """, Map("x" -> "2023-02-29")) should returnNull()
+    evaluateExpression(""" date(2023, 2, 29) """) should returnNull()
+  }
+
+  it should "return null if the date is not valid (month with 31 days)" in {
+
+    evaluateExpression(""" date(x) """, Map("x" -> "2023-06-31")) should returnNull()
```

Comment:
🔧 2023-06-31 is a month without 31 days, but the test seems to imply otherwise:

```suggestion
  it should "return null if the date is not valid (month without 31 days)" in {

    evaluateExpression(""" date(x) """, Map("x" -> "2023-06-31")) should returnNull()
```